gear/form_qc_checker/src/python/form_qc_app/error_report.py

```diff
@@ -0,0 +1,13 @@
+"""Error reporting module"""
+
+
+class ErrorReport:
```

Comment:
I'd prefer we use common mechanism for this. It doesn't have to be mine necessarily. But take a look at outputs.errors.ErrorWriter and see if it would work here

Reply:
This class is intended to retrieve error descriptions from where they are stored (TBD). When writing error metadata to Flywheel, I'm planning to use the common modules. Ignore the linting issues for now as class is not developed.